packages/@aws-cdk/aws-neptune-alpha/test/cluster.test.ts

```diff
@@ -33,8 +33,8 @@ describe('DatabaseCluster', () => {
     });
 
     Template.fromStack(stack).hasResource('AWS::Neptune::DBInstance', {
-      DeletionPolicy: 'Retain',
-      UpdateReplacePolicy: 'Retain',
+      DeletionPolicy: 'Delete',
+      UpdateReplacePolicy: 'Delete',
     });
```

Comment:
Add a testcase where defaults are picked when Deletion and UpdateReplace policy are not set and vice-versa.

Reply:
As explained in https://github.com/aws/aws-cdk/pull/25775#discussion_r1209796704, the DeletionPolicy (and UpdateReplacePolicy) for the instance(s) auto-created via the cluster prop `instances?` are expected to always use `Delete` as default. The policy specified by the user for the cluster construct should only be applied to the cluster.